src/environment/dev/app/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM composer/composer:2.5.7-bin as composer-bin
+
+FROM node:20.2.0-bullseye-slim as node
+
+FROM php:8.2.5-fpm-bullseye
+
+COPY --from=composer-bin ./composer /usr/bin/composer
```

Comment:
Please update all versions to the newest.

Reply:
updated